### PR TITLE
adding Stack impl and tests

### PR DESCRIPTION
**Pull request by kunalbohra**

### CODE REVIEW

File: stacks/Stack.test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    expect(stack.size).toBe(5);
+  });
+
+  test('peek returns the top of the stack without altering the size of stack', () => {
+    const stack = new Stack();
+    [1, 2, 3, 4, 5].forEach(stack.push, stack);
+    expect(stack.peek()).toBe(5);
+    expect(stack.size).toBe(5);
+  });
+});
```
Comment (natehermens) [inline stacks/Stack.test.js:43]: Your tests look good.  The only other edge cases I could think that might be worth having would be a 'peek' and a 'pop' on an empty stack.

Comment (natehermens) [reply stacks/Stack.test.js:43]: Your new changes look good to me
